app/theme/_common.scss

```diff
@@ -263,12 +263,11 @@ body {
     width: 100%;
 
     h2 {
+        color: $header-heading-color;
         font-size: 1.385rem;
         line-height: 1.692rem;
         margin: 0 0.5rem 0 0;
-        color: $header-heading-color;
```

Comment:
Please just normal typography elements and then override the headers using bootstrap variables or design the general panel-title.

But designing the header for just the rules thing makes absolutely no sense.